airflow/providers/cncf/kubernetes/provider.yaml

```diff
@@ -124,3 +125,212 @@ connection-types:
 task-decorators:
   - class-name: airflow.providers.cncf.kubernetes.decorators.kubernetes.kubernetes_task
     name: kubernetes
+
+config:
+  local_kubernetes_executor:
+    description: |
+      This section only applies if you are using the ``LocalKubernetesExecutor`` in
+      ``[core]`` section above
+    options:
+      kubernetes_queue:
+        description: |
+          Define when to send a task to ``KubernetesExecutor`` when using ``LocalKubernetesExecutor``.
+          When the queue of a task is the value of ``kubernetes_queue`` (default ``kubernetes``),
+          the task is executed via ``KubernetesExecutor``,
+          otherwise via ``LocalExecutor``
+        version_added: ~
+        type: string
+        example: ~
+        default: "kubernetes"
+  kubernetes_executor:
+    description: ~
+    options:
+      api_client_retry_configuration:
+        description: |
+          Kwargs to override the default urllib3 Retry used in the kubernetes API client
+        version_added: ~
+        type: string
+        example: '{ "total": 3, "backoff_factor": 0.5 }'
+        default: ""
+      logs_task_metadata:
+        description: |
+          Flag to control the information added to kubernetes executor logs for better traceability
+        version_added: 7.4.0
```

Comment:
This feels a little weird to me. I know this was just added in core, but all of these were just added into the provider. `api_client_retry_configuration` above is a good example. That was just added in 2.6.0 itself.

I wonder if a "fresh clean slate" is better?

Reply:
Makes sense.